### PR TITLE
Fix Nvidia target name

### PR DESCRIPTION
**Pull request by kwozyman**

### CODE REVIEW

**Review by Gregory-Pereira — CHANGES_REQUESTED** [review]
Seems that all the other make targets have been moved back to `image` rather than `bootc`. I do not think this change is correct, but please let me know if I am misunderstanding something

**Comment by kwozyman** [comment]
Hmm, I see in the main training/Makefile we still call "bootc" for all platforms

**Comment by Gregory-Pereira** [comment]
Got it, I wasn't checking the Makefile for the traning dir, it seems that you are right that [makefile calls it bootc](https://github.com/containers/ai-lab-recipes/blob/main/training/Makefile#L44-L48). However, all of the other Makefiles ([intel-bootc](https://github.com/containers/ai-lab-recipes/blob/main/training/intel-bootc/Makefile#L16-L17), [nvidia-bootc](https://github.com/containers/ai-lab-recipes/blob/main/training/nvidia-bootc/Makefile#L41-L42), [amd-bootc](https://github.com/containers/ai-lab-recipes/blob/main/training/amd-bootc/Makefile#L13-L14) `Makefiles` all use `image` instead of `bootc`. I would love some clarification from @rhatdan on what we want this to be.

File: training/nvidia-bootc/Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ dtk:
 		.
 
 .PHONY: image
```
Comment (rhatdan) [inline training/nvidia-bootc/Makefile:41]: This should be bootc as well, Well should have nothing I believe.

**Comment by rhatdan** [comment]
Everything should be bootc, not image.

**Comment by rhatdan** [comment]
This looks like it is fixed in main.

**Pull request closed without merging**